### DISCUSSION TITLE
Cannot override docker user while using image renovatebot/renovate

### DISCUSSION THREAD

**Discussion by ylazakovich** [renovatebot/renovate#30749] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab
image: `ghcr.io/renovatebot/renovate:38.26.0`

### Please tell us more about your question or problem

I have used GitLab bot example from documentation:
https://docs.renovatebot.com/examples/self-hosting/#gitlab-cicd-pipeline

As I understand currently we faced user conflict while running image on gitlab runners
- I tried to used `config.js` and override `"dockerUser": "1000:my_user_group_id"` (I know what group I should use here)
but 
1) console log returned 1000:0 as default but I expect overrided group
2) log file still don't exist 
3) if use RENOVATE_BASE_DIR will return ERROR about mkdir folder
All these point refer to issue with dockerUser   

Also I tried to use 
1)`RENOVATE_DOCKER_USER`: `"1000:my_user_group_id"`
> export RENOVATE_DOCKER_USER="$(id -u):my_user_group_id"

2)`RENOVATE_BINARY_SOURCE `: `docker`
3)`RENOVATE_DOCKER_CLI_OPTIONS`: `--group-add my_user_group_id`

Finally I'm calling command
> echo user - "$(id -u):$(id -g)" # 1000:0 but expect 1000:my_user_group_id
> whoami # ubuntu 

Would you please provide any information...
1) Is dockerUser work ? 
2) Is it possible to resolve this conflict without changing config of gitlab runners ? 
3) Is it possible to resolve this conflict while running image over gitlab ? 
- I mean without direct call `docker run...`

Current what I can do, it it just disable BASE_DIR and LOG_FILE
- but I want to use these features

**Comment by ylazakovich** [comment]
but I'm not sure that I'm exactly understand issue which I faced

could you please provide me answer on one question:

> I have used GitLab bot example from documentation:
> https://docs.renovatebot.com/examples/self-hosting/#gitlab-cicd-pipeline

If I run this example and I don't see log file, does it mean that it is a conflict in user ? Or might be there is other issue ?
> WARNING: renovate-log.ndjson: no matching files. Ensure that the artifact path is relative to the working directory (/builds/xxx/xxx) 
> ERROR: No files to upload

**Reply by viceice** [reply]
you should start here

https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/README.md#renovate-runner
